### PR TITLE
Update incremental build log message to show correct image tag

### PR DESCRIPTION
**Pull request by rhcarvalho**
@bparees ptal

### CODE REVIEW

**Comment by smarterclayton** [comment]
Were the tests deleted upstream, or godep just excluded them?

File: Godeps/_workspace/src/github.com/openshift/source-to-image/pkg/build/strategies/sti/sti.go [hunk 1]
```diff
@@ -143,7 +143,11 @@ func (b *STI) Build(config *api.Config) (*api.Result, error) {
 	}
 
 	if b.incremental = b.artifacts.Exists(config); b.incremental {
-		glog.V(1).Infof("Existing image for tag %s detected for incremental build", config.Tag)
+		tag := config.IncrementalFromTag
+		if len(tag) == 0 {
+			tag = config.Tag
+		}
+		glog.V(1).Infof("Existing image for tag %s detected for incremental build", tag)
```
Comment (rhcarvalho) [inline Godeps/_workspace/src/github.com/openshift/source-to-image/pkg/build/strategies/sti/sti.go]: These are the interesting bits, the rest is just the removal of test files added in the last bump by @csrwng.
The [script in source-to-image](https://github.com/openshift/source-to-image/blob/master/hack/rebase-to-origin.sh) explicitly remove test files, godep doesn't, that's the difference.
Talked with @csrwng and concluded we don't require the test files under Godeps/_workspace.

**Comment by rhcarvalho** [comment]
> Were the tests deleted upstream, or godep just excluded them?

We have a [script in source-to-image](https://github.com/openshift/source-to-image/blob/master/hack/rebase-to-origin.sh) that explicitly remove the tests.

In the last bump a couple of days ago @csrwng used vanilla `godep` and added all tests (none was in origin). He was unaware of the script ;)

If we want to always keep them, I can go ahead and change the script.

**Comment by rhcarvalho** [comment]
@mfojtik since you're the original author of the script, you might have an opinion here ;)

**Comment by bparees** [comment]
i think we keep them for other godeps?  probably better to keep them.

**Comment by rhcarvalho** [comment]
I have nothing against having them; updated the patch to include only the bump + changes I'm interested in.
@bparees ptal

**Comment by bparees** [comment]
lgtm [merge]

**Comment by mfojtik** [comment]
if godeps preserves them the script should do the same.

**Comment by rhcarvalho** [comment]
> if godeps preserves them the script should do the same.

I've updated the script already and it was merged in source-to-image ;)

**Pull request merged**